core/src/style/polylineStyle.cpp

```diff
@@ -112,29 +128,42 @@ struct PolylineStyleBuilder : public StyleBuilder {
 
     void addMesh(const Line& _line, const Parameters& _params);
 
-    void buildLine(const Line& _line, const Parameters::Attributes& _att,
-                   MeshData<PolylineVertex>& _mesh);
+    void buildLine(const Line& _line, const typename Parameters::Attributes& _att,
```

Comment:
To answer the above: Because `Paramters::Attribute` is in a dependent scope on the template class. (refer: https://rmf.io/cxx11/dependent-names-bliss/)